new/LIT6957HassabaHegg.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6957HassabaHegg" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Ḥassāba ḥǝgg</title>
```

Comment:
It would be better to shape the title section according to the scheme in the guidelines, `<title xml:lang="gez" xml:id="t1">መጽሐፈ፡ ቅዳሴ፡</title>`
         `<title xml:lang="gez" corresp="#t1" type="normalized">Maṣḥafa qǝddāse</title>`
         `<title xml:lang="en" corresp="#t1">Book of the Consecration</title>` 
that is, with Ethiopic and translation, even though approximate.

Reply:
What do you suggest as translation for Ḥassāba ḥǝgg? Is "Calculation of feasts" an acceptable approximation?

Reply:
I would translate literally, "Calculation of the law", whateve it might mean; just in case it is completely misleading, I create an additional supplied English title.